cms/templates/js/publish-xblock.underscore

```diff
@@ -101,6 +101,10 @@ var visibleToStaffOnly = visibilityState === 'staff_only';
             </a>
             </li>
         </ul>
+        <p>
+            <%- gettext("Note: Do not hide graded assignments using this feature.") %>
```

Comment:
Ok.  Gotcha.  Should we clarify the note to say not to use the feature for already released graded assignments?  For assignments that haven't been released yet, it's perfectly fine.  And we want to encourage them to use this feature for still in-development assignments, right?
`Do not hide graded assignments after they have been released.`